docs/runtime/instantiable-module.md

```diff
@@ -0,0 +1,140 @@
+
+
+## What is an Instantiable Module?
+An instantiable runtime module is, in most ways, just like any other runtime module. It defines storage items, events, dispatchable calls, and a genesis configuration. But with just a few extra bits of syntax, the module can be made instantiable which means it can be deployed multiple times in the same runtime.
+
+Some use cases:
+
+* Token chain hosts two independent cryptocurrencies.
+* Marketplace track users' reputations as buyers separately from their reputations as sellers.
+* Governance has two (or more) houses which act similarly internally.
+
+The SRML's Balances and Collective modules are good examples of real-world code using this technique.
+
+## Writing an Instantiable Module
+Writing an instantiable module is almost entirely the same process as writing a plain non-instantiable module. There are just a few places where the syntax differs. If you prefer video walkthroughs, you can see a [recording](https://www.youtube.com/watch?v=XEl59hVcyI8) of making these changes.
+
+### Configuration Trait
+```rust
+pub trait Trait<I: Instance>: system::Trait {
+	// TODO: Add other types and constants required configure this module.
+
+	/// The overarching event type.
+	type Event: From<Event<Self, I>> + Into<<Self as system::Trait>::Event>;
+}
+```
+
+### Storage Declaration
+```rust
+decl_storage! {
+	trait Store for Module<T: Trait<I>, I: Instance> as TemplateModule {
+		...
+	}
+}
+```
+
+### Declaring the Module Struct
+```rust
+decl_module! {
+	/// The module declaration.
+	pub struct Module<T: Trait<I>, I: Instance> for enum Call where origin: T::Origin {
+		...
+	}
+}
+```
+### Accessing Storage
+```rust
+<Something<T, I>>::put(something);
+```
+
+<!-- In v2.0 the T is omitted for both instantiable and non-instantiable modules
+<Something<I>>::put(something); -->
+
+### Event initialization
+```rust
+fn deposit_event<T, I>() = default;
+```
+
+### Event Declaration
+```rust
+decl_event!(
+	pub enum Event<T, I> where AccountId = <T as system::Trait>::AccountId {
+		...
+	}
+}
+```
+
+## Deploying an Instantiable Module
+
+When composing a runtime, one must take care to use the proper syntax for instantiable modules. As before it is slightly different.
+
+### Implementing the Configuration Trait
+When implementing the module's configuration trait, you need to be explicit which instance you are configuring, because in general the instances need not have identical configurations. In this snippet we've named the instance `Instance1`.
```

Comment:
Would say it more like:

"Each instance of a module needs to be individually configured. When implementing the module's configuration trait, you must specify the instance you are configuring. In the example below, we configure the instance named `Instance`."

Reply:
I think the general feeling I get is that you are making suggestions to me, rather than telling me what should be done. If it is a fact, use words like "must" and "need"

Reply:
Agree with the first comment. Not sure about the second. I used exactly those words.